File: 0935.pas
program p0935;

var x1, y1, x2, y2: integer;

begin
  
  readln(x1, y1, x2, y2);
  
  if ((x1 + y1) mod 2 = 0) and ((x2 + y2) mod 2 = 0) or ((x1 + y1) mod 2 > 0) and ((x2 + y2) mod 2 > 0) then
    print('YES')
  else
    print('NO');
  
end.

File: 0935.py
x1,  y1,  x2,  y2 = map(int, input().split())

if (x1 + y1) % 2 == 0 and (x2 + y2) % 2 == 0 or (x1 + y1) % 2 > 0 and (x2 + y2) % 2 > 0:
	print('YES')
else:
	print('NO')
